src/applications/appeals/10182/content/contestableIssues.jsx

```diff
@@ -25,9 +25,9 @@ export const ContestableIssuesLegend = ({ onReviewPage, inReviewMode }) => {
   } else {
     wrapClassNames.push('vads-u-margin-top--0');
   }
-  return (
+  return onReviewPage && inReviewMode ? null : (
```

Comment:
We don't want to show the `legend` in review mode - that's when you see the "Edit" button to modify the page. The whole title "Select the issues you’d like us to review" is weird:
* On the page within the form, it is an `H4` within the `legend`
* On the review & submit page in "review" mode, it is completely hidden because the content doesn't make sense - select the issues sounds like it wants the Veteran to do something other an review the info 
* On the review & submit page in "edit" mode, we're back to an `H4`

I guess we don't really need to use `Wrap` - it was necessary to change the `H4` into a `div` in review mode to keep the header levels logical - on the review & submit page all header levels get increased by one because the accordion headers are `H3`s. I hope that makes sense.